lib/string-checker.js

```diff
@@ -130,7 +130,12 @@ StringChecker.prototype = {
 
         this._configuredRules.forEach(function(rule) {
             errors.setCurrentRule(rule.getOptionName());
-            rule.check(file, errors);
+
+            try {
```

Comment:
Comparing typical linters with codestyle checker is like spoons to knifes. Yes, both written on nodejs, both checking your code, but jscs requires valid code and recommends using some linter before launching itself.

Each project that using JSCS using it with (eslint or jslint or jshint). Do you know a project where JSCS superseding an usual linter tool? I don't ;-)